content/blogposts/2022/release-4.3.md

```diff
@@ -0,0 +1,106 @@
+tags: [blog, release]
+slug: "release/4.3"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.3/sourcegraph-4-3-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.3/sourcegraph-4-3-hero.png
+changelogItems:
+  - description: "A \"copy path\" button has been added to file content, path, and symbol search results on hover or focus, next to the file path. The button copies the relative path of the file in the repo in the same way as the \"copy path\" button in the file and repo pages."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42721
+    category: Search
+  - description: "Unindexed search now uses the index for files that have not changed between the unindexed commit and the indexed commit. The result is faster unindexed search in general. If you are noticing issues you can disable by setting the feature flag `search-hybrid` to false."
+    url: https://github.com/sourcegraph/sourcegraph/issues/37112
+    category: Search
+  - description: "Mounted files in batch changes can now be viewed via the UI on the executions page."
+    url: https://github.com/sourcegraph/sourcegraph/pull/43180
+    category: Batch Changes
+  - description: "When the setting `batchChanges.enforceForks` is enabled, Batch Changes will now prefix the name of the fork repo it creates with the original repo's namespace in order to prevent repo name collisions."
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#forks
+    category: Batch Changes
+  - description: "Removed the experimental feature setting `showComputeComponent`. Any notebooks that used the compute component will no longer render the block. The block will be deleted from the database the next time a notebook that uses it is saved."
+    url: https://github.com/sourcegraph/sourcegraph/pull/45360
+    category: Notebooks
+  - description: "We'll be making a breaking change to the search query syntax with the upcoming 4.5 release in Februrary. We will be deprecating the fields `repohasfile` and `repohascommitafter` in favor of `repo:has.path()`."
+    url: https://github.com/sourcegraph/sourcegraph/issues/45625
+    category: Code search
+---
+
+<Badge link="/code-search" text="Code Search" color="cerise" size="small" />
+
+#### Instantly navigate repositories, files, and symbols by name with fuzzy finder 
+
+[Fuzzy finder](https://docs.sourcegraph.com/code_search/explanations/features) helps you quickly navigate to a repository, symbol, or file. To open the fuzzy finder, press 'Cmd+K' (macOS) or 'Ctrl+K' (Linux/Windows) from any page, and each tab has a dedicated shortcut:
+
+- Repos: Cmd+I (macOS), Ctrl+K (Linux/Windows)
+- Symbols: Cmd+O (macOS), Cmd+Shift+O (macOS Safari), Ctrl+O (Linux/Windows)
+- Files: Cmd+P (macOS), Ctrl+P (Linux/Windows)
+
+Fuzzy finder is an experimental feature, and we welcome your feedback and input.
```

Comment:
Do you want to link to where folks can go to give feedback?

Reply:
```suggestion
Fuzzy finder is an experimental feature, and we welcome your feedback and input. You can [tweet at us](https://twitter.com/sourcegraph) or [send us an email](mailto:feedback@sourcegraph.com).
```